main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file).items()
+
+    for player_name, player_data in data:
+        race_data = player_data["race"]
+        guild_data = player_data["guild"]
+
+        race, created = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data["description"]}
+        )
+
+        for skill_data in player_data["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                defaults={"bonus": skill_data["bonus"]},
+                race=race,
+            )
+
+        if guild_data:
```

Comment:
Define `guild = None` first

Reply:
Pls, give me some explanation. Do not understand